presto-orc/src/main/java/com/facebook/presto/orc/OrcPredicate.java

```diff
@@ -41,6 +42,6 @@ public boolean matches(long numberOfRows, Map<Integer, ColumnStatistics> statist
     // Returns a map from column index to Filter.
     default Map<Integer, Filter> getFilters()
     {
-        throw new UnsupportedOperationException();
+        return ImmutableMap.of();
```

Comment:
It doesn't seem correct to have default implementation for getFilters return empty map. Throwing unsupported exception seemed better. Is there any particular reason you made this change instead of add getFilters implementation to TRUE instance?